### ISSUE TITLE
UI focus indicator missing for keyboard navigation

### ISSUE THREAD

**Issue by joeyparrish** [google/shaka-player#2863]
**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
Yes

**What version of Shaka Player are you using?**
v2.5.x and v3.0.x and master

**Can you reproduce the issue with our latest release version?**
Yes

**Can you reproduce the issue with the latest code from `master`?**
Yes

**Are you using the demo app or your own custom app?**
Demo

**What browser and OS are you using?**
ChromeOS 85 stable

**What are the manifest and license server URIs?**
"Angel One" or any demo content

**What did you do?**

Play the content, then use the tab key to navigate to the UI controls

**What did you expect to happen?**
There should be some visual indication of what has keyboard focus

**What actually happened?**
No indicator appears

**Comment by CodeDotJS** [comment]
I'm working on it @joeyparrish

**Comment by joeyparrish** [comment]
Thanks!

**Comment by CodeDotJS** [comment]
@joeyparrish can you please add the gsoc label over this issue?

**Comment by joeyparrish** [comment]
I have a fix for this in review now.  Thanks, @CodeDotJS, for the PR that started the conversation and investigation into this.
